### PR TITLE
🤖 stuff

### PR DESCRIPTION
**Pull request by skylarbarrera**
fixes:
- LP Sheet height + scrolling ( should be able to scroll all of the content)
- swap currency list should still show when tapping the info or rainbow verifies buttons (before it was showing a loading state), fav button should work( its slow). Query should still reset when going back to the sheet(select input and go back to select a new one)
- L2 disclaimer should look the same as iOS, component in the l2 expanded states
- Walletconnect should show full sheet + gas (it was cut off before) make sure to check connect sign + tx
- Switching the lists on discover should not impact the display of the DPI button( before the trading at $x was being cut off)
- last NFT family should be on top of the swap/send fabs vs underneath when collapsed, check dame.eth
- Transaction list should have the same spacing as iOS, before it was not centered

### CODE REVIEW

**Review by osdnk — DISMISSED** [review]

**Review by brunobar79 — DISMISSED** [review]

File: src/components/value-chart/Chart.js [hunk 1]
```diff
@@ -215,7 +215,7 @@ export default function ChartWrapper({
             <ChartPath
               fill="none"
               gestureEnabled={!fetchingCharts && !!throttledData}
-              hapticsEnabled
+              hapticsEnabled={ios}
```
Comment (brunobar79) [inline src/components/value-chart/Chart.js:218]: no

**Review by brunobar79 — APPROVED** [review]

**Review by osdnk — APPROVED** [review]
I feel I was reading this already a few times and this is good. Looks like no-op on iOs

**Comment by osdnk** [comment]
I opt for deferring QA for later

**Review by ibrahimtaveras00 — CHANGES_REQUESTED** [review]
Only issue is that tapping fav button doesn't work on swap currency list, otherwise all other issues are fixed

**Review by ibrahimtaveras00 — APPROVED** [review]
👍🏽

**Review by terrysahaidak — COMMENTED** [review]
LGTM

**Pull request merged**